test/rmode/config.yml

```diff
@@ -0,0 +1 @@
+flags: [--float]
```

Comment:
Let's not create a new regressions folder just yet. Please put all of these into the floats folder.